css-layout-classes/index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <link rel="stylesheet" href="styles.css">
+    <title>Layout</title>
+  </head>
+
+  <body>
+
+    <div id="body">
+      <div id="title">
+        <h1>Choose your character!</h1>
+      </div>
+
+
+      <div class="row">
+
+        <div class="container" id="container-one">
+
+          <div id="image">
```

Comment:
This should be a class related to a column. Since you want this to take up 1/3rd of the page, an appropriate name would be `column-third`